apps/fm/js/fm.js

```diff
@@ -350,7 +357,27 @@ var frequencyDialer = {
       document.body.addEventListener('touchend', fd_body_touchend, false);
     }
 
+    function fd_key(event) {
+      if (event.type !== 'keypress') {
+          return;
+      }
+
+      switch (event.keyCode) {
```

Comment:
Lets just use if else here:
```
if (event.keyCode === KeyEvent.DOM_VK_UP) {
...
} else if (event.keyCode === KeyEvent.DOM_VK_DOWN) {
...
} else {
  return; 
}
```

^ Return in else to avoid calling lines 374-375